draft-dekater-scion-pki.md

```diff
@@ -482,18 +482,14 @@ The recommended **maximum validity period** of a sensitive voting certificate is
 
 All certificates used in the SCION control-plane PKI are X.509 v3 certificates. However, the SCION specification is in some places more restrictive. This section defines these additional constraints and conditions compared to {{RFC5280}} for each type of SCION control-plane PKI certificate.
 
-**Note**: The settings for the SCION-specific constraints and conditions are based on the SCION open-source implementation [scionproto](https://github.com/scionproto/scion/). Adjusting these settings to the requirements of a customer implementation may be possible and is allowed.
 
 ### Basic Fields: SCION-Specific Constraints and Conditions
 
 This section briefly describes the fields of the SCION control-plane PKI certificates based on X.509. These fields are relevant for each SCION certificate used in the control plane, regardless of the certificate type. For detailed descriptions of the full generic format of X.509 v3 certificates, see {{RFC5280}} and [X509](https://handle.itu.int/11.1002/1000/13031), clause 7.2. Additionally, the section lists the SCION-specific constraints and conditions compared to {{RFC5280}}, per certificate field.
 
 `TBSCertificate` sequence: Contains information associated with the subject of the certificate and the CA that issued it. It includes the following fields:
 
-- `version` field: Describes the version of the encoded certificate.
-
-  - **SCION constraints**: "v1" and "v2" are not allowed.
-  - **Additional conditions and remarks**: MUST be set to "v3" (as extensions are used and mandatory in SCION).
+- `version` field: Describes the version of the encoded certificate. It MUST be set to "v3" (as extensions are used and mandatory in SCION).
```

Comment:
"...used and mandatory in " How about simply "...are required by"? (Potentially "required" in upper-case, even.

Reply:
Done